src/Cache_Command.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
 /**
- * Manage the object cache.
+ * The cache command manipulates the WP Object Cache. Codex: https://codex.wordpress.org/Class_Reference/WP_Object_Cache.
```

Comment:
The short description needs to be 80 characters or less, so I think it makes sense to drop the link.

Instead, the link can go in the extended description (next blog of text) as a markdown link (so it renders as a link on the website).

Is there a rephrasing "The cache command manipulates the WP Object Cache." so that we don't start every command description with "The" ?